app/jobs/jira-ticket.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+    "fields": {
```

Comment:
why u need this?

Reply:
this file is proto type of create jira ticket useing this file json in create api